src/bbs_plus.rs

```diff
@@ -88,7 +89,7 @@ wasm_impl!(
 );
 
 #[wasm_bindgen(js_name = sign)]
-pub fn bbs_sign(request: JsValue) -> Result<JsValue, JsValue> {
+pub async fn bbs_sign(request: JsValue) -> Result<JsValue, JsValue> {
```

Comment:
nice! This is the first time I'm actually seeing async used in rust. Indy SDK never got upgraded to it after it was announced.